koku/masu/processor/ocp/ocp_report_db_cleaner.py

```diff
@@ -60,8 +60,7 @@ def purge_expired_line_item(self, expired_date, provider_uuid=None, simulate=Fal
             err = "Parameter expired_date must be a datetime.datetime object."
             raise OCPReportDBCleanerError(err)
 
-        with ReportingCommonDBAccessor() as reporting_common:
-            column_map = reporting_common.column_map
+        column_map = REPORT_COLUMN_MAP
```

Comment:
Same here.  Remove column_map from `OCPReportDBAccessor` constructor.

Reply:
@kholdaway there are over 500 hundred references to column_map in our codebase

I estimate your suggestion will add at least two days to the delivery of this story.

Reply:
We can chat about it tomorrow if you would like but I don't agree it's that hard.

Reply:
updated